tests/ui/docker-compose.selenium.yml

```diff
@@ -15,7 +15,7 @@ x-env-mapping: &env
     - TERM=xterm-256color
     - UITEST_FXA_EMAIL
     - UITEST_FXA_PASSWORD=uitester
-    - MOZ_HEADLESS=1
+    - MOZ_HEADLESS
```

Comment:
Is this unsetting this or what is this doing? This isn't clear to be just from looking at it.

Reply:
Just getting it from the ENV and passing it in